widget/button.go

```diff
@@ -182,11 +313,51 @@ func (b *buttonRenderer) Refresh() {
 	canvas.Refresh(b.button.super())
 }
 
+func alignedPosition(align ButtonAlign, padding, objectSize, layoutSize fyne.Size) (pos fyne.Position) {
```

Comment:
nit: should be below the renderer methods

Reply:
thanks, not surprised I missed one in the reshuffle